models/articles.models.js

```
@@ -82,3 +86,24 @@ exports.insertComment = (username, body, article_id) => {
       });
   });
 };
+
+exports.updateVotes = (article_id, inc_votes) => {
+  return checkArticleExists(article_id)
+    .then(() => {
+      return checkValidDataType(inc_votes);
```

you dont need to manually check that the data type is valid because if its invalid then PSQL should throw an error

---

Noted and makes sense when thinking about it - I think I just forced using the util function to check before processing the model.